aws/table_aws_ec2_capacity_reservation.go

```diff
@@ -0,0 +1,215 @@
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "availability_zone_id",
+				Description: "The availability zone ID of the capacity reservation.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "available_instance_count",
+				Description: "The remaining capacity. Indicates the number of instances that can be launched in the capacity reservation.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "create_date",
+				Description: "The date and time at which the capacity reservation was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "ebs_optimized",
+				Description: "Indicates whether the capacity reservation supports EBS-optimized instances.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "end_date",
+				Description: "The date and time at which the capacity reservation expires.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "end_date_type",
+				Description: "Indicates the way in which the capacity reservation ends. A capacity reservation can have one of the following end types: 'unlimited', 'limited'.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "ephemeral_storage",
+				Description: "Indicates whether the capacity reservation supports instances with temporary, block-level storage.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "instance_match_criteria",
+				Description: "Indicates the type of instance launches that the capacity reservation accepts. The options include: 'open', 'targeted'.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "instance_platform",
+				Description: "The type of operating system for which the capacity reservation reserves capacity.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "owner_id",
+				Description: "The ID of the AWS account that owns the capacity reservation.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "start_date",
+				Description: "The date and time at which the capacity reservation was started.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "tenancy",
+				Description: "Indicates the tenancy of the capacity reservation. A capacity reservation can have one of the following tenancy settings: 'default', 'dedicated'.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "total_instance_count",
+				Description: "The total number of instances for which the capacity reservation reserves capacity",
+				Type:        proto.ColumnType_INT,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("CapacityReservationId"),
+			},
+			{
+				Name:        "tags",
```

Comment:
Format tags as map[string]string{}, For reference check aws_tagging_resource table.